examples/pinn/eikonal.py

```diff
@@ -0,0 +1,459 @@
+"""
+Title: Neural Eikonal Solver
```

Comment:
Thanks for your comments and reply!
I was also thinking about "quick keras recipes" but alternatively since eikonal is widely used in computer vision if I adapt the example to a computer vision context (e.g. to Shape-from-Shading problem), do you think that "computer vision" could be a good fit?

Reply:
I say we put it in quick keras recipes and focus on the concept of PINNs!
I think we should emphasize that in PINNs you simply add an L2 loss to enforce the implicit physics constraints of the system.